src/js/View.js

```diff
@@ -425,12 +425,12 @@ View = (function() {
             else if (view.mode==View.SELECT) {
                 view.selectStartCoo = {x: view.dragx, y: view.dragy};
             }
-            return false; // to disable text selection
+            return true; // false disables default browser behaviour like possibility to touch hold for context menu. To disable text selection use css user-select: none instead of putting this value to false
         });
 
         //$(view.reticleCanvas).bind("mouseup mouseout touchend", function(e) {
-        $(view.reticleCanvas).bind("click mouseout touchend", function(e) { // reacting on 'click' rather on 'mouseup' is more reliable when panning the view
-            if (e.type==='touchend' && view.pinchZoomParameters.isPinching) {
+        $(view.reticleCanvas).bind("click mouseout touchend touchcancel", function(e) { // reacting on 'click' rather on 'mouseup' is more reliable when panning the view
```

Comment:
This is just some handling of an event that almost never happens: touchcancel. 
https://developer.mozilla.org/en-US/docs/Web/API/Element/touchcancel_event
Probably still a good idea to handle this event, to make sure Aladin Lite doesn't get stuck in the incorrect state when a touchcanel event finally happens. For example, continuing to be in the moving state after a touchcancel when touchmove events have stopped coming